unity-client/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/LoadWrapper/LoadWrapper_NFT.cs

```diff
@@ -37,10 +40,19 @@ public override void Load(string src, System.Action<LoadWrapper> OnSuccess, Syst
 
             loaderController.OnLoadingAssetSuccess += CallOnComponentUpdated;
 
-            loaderController.OnLoadingAssetSuccess += () => OnSuccess(this);
-            loaderController.OnLoadingAssetFail += () => OnFail(this);
+            assetUrl = src;
 
-            loaderController.LoadAsset(src, true);
+            if (CommonScriptableObjects.rendererState)
```

Comment:
This wouldn't be like this?
```suggestion
            if (CommonScriptableObjects.rendererState.Get())
```

Reply:
`BaseVariable<T>`  has an implicit cast to `T` , boolean in this case, but I'll change it anyway to avoid any confusion